test/lib/nvme/unit/nvme_ctrlr_cmd_c/nvme_ctrlr_cmd_ut.c

```diff
@@ -459,10 +459,18 @@ static void
 test_abort_cmd(void)
 {
 	struct spdk_nvme_ctrlr	ctrlr = {};
+	struct spdk_nvme_qpair	qpair = {};
+	ctrlr.curr_abort_count = 0;
 
 	verify_fn = verify_abort_cmd;
 
-	nvme_ctrlr_cmd_abort(&ctrlr, abort_cid, abort_sqid, NULL, NULL);
+	qpair.id = abort_cid;
```

Comment:
The queue pair ID should not be assigned a command ID (those are different, unrelated things). I believe abort_sqid is what you want.